src/Vendoring/OpenTelemetry.Instrumentation.EventCounters/EventCountersInstrumentationEventSource.cs

```diff
@@ -0,0 +1,58 @@
+// <copyright file="EventCountersInstrumentationEventSource.cs" company="OpenTelemetry Authors">
```

Comment:
There is no plan on taking this package stable AFAIK. cc @reyang 

My vote is we take option 1 in https://github.com/dotnet/aspire/issues/2819 under `OpenTelemetry.Instrumentation.EventCounters`. Let's remove these metrics all together. They aren't very friendly to work with anyway, with the `ec_` prefix, and the name gets truncated randomly.

Instead, we should push on the underlying ADO.NET providers to emit OTel compliant metrics using the built-in Meter APIs.

Reply:
They can always add it I guess?

Reply:
> There is no plan on taking this package stable AFAIK. cc @reyang

@reyang is that true?

Reply:
> They can always add it I guess?

Yes, that is my thinking. If the user wants these, they can add it. But we shouldn't be doing it by default anymore.

Reply:
> > There is no plan on taking this package stable AFAIK. cc @reyang
> 
> @reyang is that true?

True.

Reply:
> Instead, we should push on the underlying ADO.NET providers to emit OTel compliant metrics using the built-in Meter APIs.

💯, this would also align with the OpenTelemetry aggregation temporality and collection cycle